### ISSUE TITLE
[request] consumption rate in machines count

### ISSUE THREAD

**Issue by Trol1face** [Shad0wlife/DSP_AssemblerUI#13]
Like how many assemblers of every input item need to fulfill this recipe

**Comment by AZV-EU** [comment]
> Like how many assemblers of every input item need to fulfill this recipe

This isn't the mod for that, you should look into a calculator website/mod that does this for you beforehand I think

**Comment by Shad0wlife** [comment]
Agreed, it's unlikely to be feasible to make a satisfactory (heh) UI for that ingame, because it's likely you'll need more than 1 stage of that to properly plan. So using a calculator for that is the best way I think.
